lib/libutee/tee_api.c

```diff
@@ -392,6 +394,13 @@ void TEE_GetREETime(TEE_Time *time)
 
 void *TEE_Malloc(uint32_t len, uint32_t hint)
 {
+	/*
+	 * return a known non-NULL invalid pointer when the
+	 * requested size is zero
+	 */
```

Comment:
also move this description to `TEE_NULL_SIZED_VA` definition.

Reply:
done and tag added too